tests/projects/test_project.py

```diff
@@ -880,6 +880,45 @@ def test_project_ops():
     assert run.output("y") == 4  # = x * 2
 
 
+@pytest.mark.parametrize(
+    "workflow_path,exception",
+    [
+        (
+            "./",
+            pytest.raises(
+                ValueError,
+                match=str(
+                    "Invalid workflow_path, please provide a valid path to a python file"
+                ),
+            ),
+        ),
+        (
+            "https://test",
+            pytest.raises(
+                ValueError,
+                match=str(
+                    "Invalid workflow_path, please provide a valid path to a python file"
+                ),
+            ),
+        ),
+        (
+            "",
+            pytest.raises(
+                ValueError,
+                match=str(
+                    "Invalid workflow_path, please provide a valid path to a python file"
+                ),
+            ),
+        ),
+        ("https://test.py", does_not_raise()),
```

Comment:
can add another one of an existing local file (there are a couple in the assets dir)